docs/features/scripts.md

```diff
@@ -0,0 +1,50 @@
+---
+id: scripts
+title: Scripts support
+---
+
+### Working with a script
+
+Whenever, Metals opens a script file with `*.sc` extension, but only when it's
+not `*.worksheet.sc` or a build server is not already started for it, users will
+be prompted to choose whether they want to use [Ammonite](http://ammonite.io/)
+or [ScalaCli](https://scala-cli.virtuslab.org/) for using within the it.
```

Comment:
Just a nit, but I never see `ScalaCLI` all as one word like this. Would it be more accurate to say `Scala CLI`?

Reply:
Actually it's `Scala CLI` based on what we wrote on the website :sweat_smile:

Reply:
Changed everywhere I think!